app/controllers/setup/finishing-touches.js

```diff
@@ -1,17 +1,71 @@
 import Controller from '@ember/controller';
+import {isEmpty} from '@ember/utils';
 import {inject as service} from '@ember/service';
 import {task} from 'ember-concurrency';
+import {tracked} from '@glimmer/tracking';
+
+const THEME_PROPERTIES = {
+    casper: ['description', 'color', 'coverImage'],
+    edition: ['description', 'color', 'coverImage'],
+    dawn: ['description', 'color', 'icon'],
+    dope: ['description', 'color', 'logo'],
+    bulletin: ['description', 'color', 'logo'],
+    alto: ['description', 'color', 'logo'],
+    journal: ['description', 'color', 'logo'],
+    wave: ['description', 'color', 'logo', 'coverImage'],
+    edge: ['description', 'color', 'logo'],
+    ease: ['description', 'color', 'logo', 'coverImage'],
+    ruby: ['description', 'color', 'logo', 'coverImage'],
+    london: ['description', 'color', 'logo'],
+    digest: ['description', 'color', 'logo']
+};
 
 export default class SetupFinishingTouchesController extends Controller {
     @service modals;
     @service router;
     @service settings;
+    @service store;
     @service themeManagement;
 
+    @tracked descriptionVisible;
+    @tracked colorVisible;
+    @tracked logoVisible;
+    @tracked iconVisible;
+    @tracked coverImageVisible;
+
+    themes = null;
+
+    // Default properties to display
+    themeProperties = ['description', 'color', 'coverImage'];
+
+    constructor() {
+        super(...arguments);
+        this.initThemeProperties.perform();
```

Comment:
This is a controller not a component so we don't really want to use a constructor here because controllers are singletons and we can't guarantee when the instance will be created.

I think the better approach would be to move the theme loading to the route's `model()` hook so we know themes will be available before the template is rendered:

```
model() {
   return this.store.findAll('theme');
}
```

That will allow us to remove everything in the controller except for a couple of getters, eg:

```
get activeTheme() {
    return this.store.peekAll('theme').findBy('active');
}

get themeProperties() {
    const themeProperties = THEME_PROPERTIES[this.activeTheme.name] || ['description', 'color', 'coverImage'];

    // converts ['color', 'description'] to {color: true, description: true} for easier use in template
    return Object.fromEntries(themeProperties.map(prop => [prop, true]));
}
```

in the template you can then change the conditionals to be `{{#if this.themeProperties.description}}`, {{#if this.themeProperties.color}}` etc